Objects/unicodeobject.c

```diff
@@ -5869,6 +5869,23 @@ PyUnicode_AsUTF16String(PyObject *unicode)
     return _PyUnicode_EncodeUTF16(unicode, NULL, 0);
 }
 
+_PyUnicode_Name_CAPI *
```

Comment:
Is there a reason why this function is not marked as "static"?

Reply:
Oh, I didn't see that it's exposed in Include/internal/pycore_ucnhash.h. Ok, fine.